.gitignore

```diff
@@ -19,3 +19,6 @@ node_modules
 
 # macOS
 .DS_Store
+
+# virtualenv
+env/
```

Comment:
Please remove this - it's specific to the way you run the code. I'd advise putting things like this in a global `.gitexclude` file.